[BUGS-4199] Document phantomjs workaround and unsupported. (#6936)
pantheon-systems/documentation: source/content/external-libraries.md

If you encounter this error, remove the offending `quotes` property from the CSS.

## PhantomJS (Unsupported)

In its own words, [PhantomJS](https://github.com/ariya/phantomjs/) is a headless WebKit with JavaScript API. It has fast and native support for various web standards: DOM handling, CSS selector, JSON, Canvas, and SVG.

> However, PhantomJS development [has been suspended until further notice](https://github.com/ariya/phantomjs/issues/15344). Pantheon keeps the following binaries but they may be removed in the future.

Reviewer: How about this phrasing instead:
>However, PhantomJS development [has been suspended until further notice](https://github.com/ariya/phantomjs/issues/15344). Pantheon continues to include the following binaries; they may be removed in the future.

Reviewer 2: +1 I like @CdrMarks' phrasing.

```suggestion
However, PhantomJS development [has been suspended until further notice](https://github.com/ariya/phantomjs/issues/15344). While Pantheon continues to include the following binaries, they may be removed in the future.
```

- PhantomJS (1.7.0) is located at `/srv/bin/phantomjs` on your application container.
- PhantomJS (2.1.1) is located at `/srv/bin/phantomjs-2.1.1` on your application container.

> Recently, PhantomJS started erroring on domains with Let's Encrypt. A known workaround for this is to ignore SSL certs error using the following option: `--ignore-ssl-errors=yes`

Reviewer 2: I think this is more grammatically correct: 

```suggestion
Recently, PhantomJS started erroring on domains with Let's Encrypt. A known workaround for this is to ignore SSL certificate errors using the following option: `--ignore-ssl-errors=yes`
```
